src/conic_form.jl

```diff
@@ -20,13 +31,21 @@ function +(c::ConicObj, d::ConicObj)
     if !haskey(new_obj, var)
       new_obj[var] = d[var]
     else
-      # .+ does not preserve sparsity
+      # .+ does not preserve sparsity 
       # need to override behavior
-      if size(new_obj[var]) == size(d[var])
-        new_obj[var] = new_obj[var] + d[var]
+      #for i in 1:2
```

Comment:
can you do this with a for loop? also, make it more clear (with at least a comment) what you're looping over.